src/entries/popup/components/WarningInfo/walletGroupWipeWarningInfo.tsx

```diff
@@ -0,0 +1,160 @@
+import React, { useState } from 'react';
+
+import { i18n } from '~/core/languages';
+import {
+  Box,
+  Button,
+  Column,
+  Columns,
+  Inline,
+  Row,
+  Rows,
+  Separator,
+  Stack,
+  Symbol,
+  Text,
+} from '~/design-system';
+import { Lens } from '~/design-system/components/Lens/Lens';
+
+import { Checkbox } from '../Checkbox/Checkbox';
+import {
+  IconAndCopyItem,
+  IconAndCopyList,
+} from '../IconAndCopyList.tsx/IconAndCopyList';
+
+const t = (s: string) =>
+  i18n.t(s, { scope: 'settings.privacy_and_security.wallets_and_keys' });
+
+interface WarningInfoProps {
+  iconAndCopyList: IconAndCopyItem[];
+  onProceed: () => void;
+  testId?: string;
+}
+
+export default function WarningInfo({
+  iconAndCopyList,
```

Comment:
maybe this don't have to be a prop either, seems like a very specific component no need to abstract it, or change the name and abstract to use in the other similar screens?